src/wwwroot/template-editor/adapterService.js

```diff
@@ -340,40 +353,10 @@
     function getSettings(idCampaign, idTemplate) {
       traceAdapterCall(getSettings, arguments);
       var msEditorSettings = {
-        language: 'es',
-        sharedSocialNetworks: [{
```

Comment:
Will it work or do you need to return an empty array for _sharedSocialNetworks_?